### PR TITLE
Adding the overall system endpoint

### PR DESCRIPTION
**Pull request by LanternNassi**
# Description

This endpoint is aimed at providing the admin with information about apps , users , projects , databases 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Trello Ticket ID

https://trello.com/c/895PI9eT/1646-admin-create-a-system-overal-summary-endpoint

## How Can This Be Tested?

This endpoint can be tested by hiting the `/system_summary` endpoint to get the information

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by MutegekiHenry — APPROVED** [review]
LGTM

File: app/controllers/system_status.py [hunk 1]
```diff
@@ -44,3 +49,47 @@ def get(self):
...
+
+        return dict(status='success' , data={
+            'Users' : {
+                'total_count' : user_count,
+                'verified' : verified_users,
+                'beta_users' : beta_users
+            },
+            'Projects' : {
+                'total_count' : project_count,
+                'active' : active_projects
```
Comment (Mubangizi) [inline app/controllers/system_status.py:85]:
```suggestion
                'disabled' : disabled_projects
```

File: app/controllers/system_status.py [hunk 2]
```diff
@@ -44,3 +49,47 @@ def get(self):
...
+    def get(self):
+
+        # Users 
+        user_count = User.query.count()
+        verified_users = User.query.filter(User.verified==True).count()
+        beta_users = User.query.filter(User.is_beta_user==True).count()
+
+        #Projects
+        project_count = Project.query.count()
+        active_projects = Project.query.filter(Project.disabled == False).count()
```
Comment (Mubangizi) [inline app/controllers/system_status.py:66]:
```suggestion
        disabled_projects = Project.query.filter(Project.disabled == True).count()
```

**Review by Mubangizi — CHANGES_REQUESTED** [review]
Great work @LanternNassi, but since disabled projects are the minor figure, thats what you are supposed to return, also the term `"active"` may be unclear when referring to projects.

File: app/controllers/system_status.py [hunk 3]
```diff
@@ -44,3 +49,47 @@ def get(self):
...
+                'total_count' : project_count,
+                'disabled' : disabled_projects
+            },
+            'Apps' : {
+                'total_count' : app_count,
+            },
+            'Databases' : {
+                'total_count' : database_count,
+                'mysql' : mysql_database_count,
+                'postgres' : database_count - mysql_database_count
```
[the remark this answers is not in the document]
Comment (Mubangizi) [reply app/controllers/system_status.py:93]: I know it doesn't make sense but they do exist

**Review by Mubangizi — APPROVED** [review]
Awesome stuff @LanternNassi

**Pull request merged**